src/Microsoft.TemplateEngine.Abstractions/IBaselineInfo.cs

```diff
@@ -1,14 +1,25 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#nullable enable
+
 using System.Collections.Generic;
 
 namespace Microsoft.TemplateEngine.Abstractions
 {
+    /// <summary>
+    /// Defines baseline information.
```

Comment:
I don't know what baseline information means. Should we be a bit more descriptive here?

Reply:
I added short explanation  `Baseline configuration is a set of predefined template parameters and their values.`, however this term is quite well known.